hatch/src/cli/commands/build.rs

```diff
@@ -17,57 +16,65 @@ impl<'build> Build {
       name: "build"
     }
   }
+
+  pub fn execute(&self, project: &Project) -> HatchResult<()> {
+    task::generate_assets(&project).with_context(|e| {
+      format!("Failed to generate assets : {}", e)
```

Comment:
In line with the changes I suggested in the implementation of the command::execute method, can we then make this error message ```format!("asset generation failed : `{}`", e)```?

Reply:
Works for me.